python/tests/serializers/test_validation_error_shape.py

```
@@ -0,0 +1,174 @@
+def test_classmethod_field_validator_reports_failures():
+    """A dropped validator fails open — the invalid value would be accepted."""
+
+    class WithClassmethod(Serializer):
+        a: str
+
+        @field_validator("a")
+        @classmethod
+        def reject(cls, v: str) -> str:
+            raise ValueError(f"nope: {v}")
+
+    with pytest.raises(Exception) as exc_info:
```

Ruff (0.16.1)

[warning] `pytest.raises(Exception)` is too broad, set the `match` parameter or use a more specific exception

(PT011)